scipy/stats/tests/test_relative_risk.py

```diff
@@ -0,0 +1,46 @@
+
+import pytest
+import numpy as np
+from numpy.testing import assert_allclose
+from scipy.stats import relative_risk
+
+
+# Test just the calculation of the relative risk, including edge
+# cases that result in a relative risk of 0, inf or nan.
+@pytest.mark.parametrize(
+    'exposed_cases, exposed_total, control_cases, control_total, expected_rr',
+    [(1, 4, 3, 8, 0.25 / 0.375),
+     (0, 10, 5, 20, 0),
+     (0, 10, 0, 20, np.nan),
+     (5, 15, 0, 20, np.inf)]
+)
+def test_relative_risk(exposed_cases, exposed_total,
+                       control_cases, control_total, expected_rr):
+    result = relative_risk(exposed_cases, exposed_total,
+                           control_cases, control_total)
+    assert_allclose(result.relative_risk, expected_rr, rtol=1e-13)
+
+
+def test_relative_risk_confidence_interval():
```

Comment:
because this is (in some sense) decoupled from the `relative_risk` we may want tests for `0` counts as well as `nan`/`inf`.

Reply:
Done.